test/e2e/core/styleParsing.css

```diff
@@ -10,7 +10,7 @@ svg path.fclass {
   stroke: blue;
 }
 
-svg.web-map.style-scope path.fclass._1 {
+svg.mapml-web-map.style-scope path.fclass._1 {
```

Comment:
I'm not sure how/where this SVG selector is used (it seems it's not to be in use atm). Should it also change to match the new `mapml-web-map` class, or do I just revert this change?

Reply:
I think that style-scope stuff is a holdover from the polymer days, but not sure.  I believe that was how they pretended that shadow dom was real (they called it "shady dom").  I might just delete the whole style if it's not in use.

Reply:
Okay, I reverted the SVG selector change just in case. Feel free to merge.